notebooks/classification/mcmc_bnn_makemoons.ipynb

```diff
@@ -0,0 +1,462 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        logits = model.apply(params, X).ravel()
</pre><p>It would be good to add small comments. like, model.apply return N, 1, ... and we flatten it to ..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nipunbatra/bnn-zoo/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>